controllers/etcd_controller.go

```diff
@@ -414,59 +357,90 @@ func (r *EtcdReconciler) delete(ctx context.Context, etcd *druidv1alpha1.Etcd) (
 
 func (r *EtcdReconciler) reconcilePodDisruptionBudget(ctx context.Context, logger logr.Logger, etcd *druidv1alpha1.Etcd, values map[string]interface{}) error {
 	logger.Info("Reconcile PodDisruptionBudget")
-	pdb := &policyv1beta1.PodDisruptionBudget{}
-	err := r.Get(ctx, types.NamespacedName{Name: etcd.Name, Namespace: etcd.Namespace}, pdb)
-
-	if err == nil {
-		// pdb already exists, claim it
-
-		selector, err := metav1.LabelSelectorAsSelector(etcd.Spec.Selector)
-		if err != nil {
-			logger.Error(err, "Error converting etcd selector to selector")
-			return err
-		}
 
-		logger.Info("Claiming pdb object")
-		_, err = r.claimPodDisruptionBudget(ctx, etcd, selector, pdb)
+	// Find out k8s version
+	targetCoreClient, err := k8s.NewForConfig(r.Config)
```

Comment:
can you move this to a utility function in utils/k8s_version.go? So essentially create the following function:
```go
func GetTargetK8sVersion(config *rest.Config) (*semver.Version, error) {
}
```

Reply:
Done